mirdata/datasets/da_tacos.py

```diff
@@ -0,0 +1,568 @@
+                "tags": self.tags,
+            }
+        )
+
+
+def load_cens(path):
+    """Load da_tacos cens features from a file
+
+    Args:
+        path(str or file-like): File-like object or path to features file
+
+    Returns:
+        np.array: cens features
+
+    """
+    return dd.io.load(path)["chroma_cens"]
+
+
+def load_crema(path):
+    """Load da_tacos crema features from a file
+
+    Args:
+        path(str or file-like): File-like object or path to features file
+
+    Returns:
+        np.array: crema features
+
+    """
+    return dd.io.load(path)["crema"]
+
+
+def load_hpcp(path):
+    """Load da_tacos hpcp features from a file
+
+    Args:
+        path(str or file-like): File-like object or path to features file
+
+    Returns:
+        np.array: hpcp features
+
+    """
+    return dd.io.load(path)["hpcp"]
+
+
+def load_key(path):
+    """Load da_tacos key features from a file.
+
+    Args:
+        path(str or file-like): File-like object or path to features file
+
+    Returns:
+        dict: key
+
+    Examples:
+        {'key': 'C', 'scale': 'major', 'strength': 0.8449875116348267}
+
+    """
+    return dd.io.load(path)["key_extractor"]
+
+
+def load_madmom(path):
+    """Load da_tacos madmom features from a file
+
+    Args:
+        path(str or file-like): File-like object or path to features file
+
+    Returns:
+        dict: madmom features
+
+    Examples:
+        {
+           "novfn":"array("[0.01775683, 0.00553825, 0.00302445, "...", 0.0027212, 0.00570413, 0.01260976]")",
+           "onsets":"array("[ 47, 116, 187, ... 11568, 11610, 11649 ]")",
+           "snovfn":"array("[ 0., 0., 0., "...", 0., 0., 0.]")",
+           "tempos":"array("[ [5.94059406e+01, 2.07677787e-01], [1.20000000e+02, 1.38685472e-01],
+                              [2.40000000e+02, 1.24457522e-01], [4.76190476e+01, 1.17567189e-01],
+                              [7.89473684e+01, 1.05035187e-01], [6.81818182e+01, 5.60316640e-02],
+                              [4.44444444e+01, 4.91360210e-02], [5.26315789e+01, 4.55353848e-02],
+                              [4.28571429e+01, 3.97281834e-02], [9.52380952e+01, 3.47324676e-02],
+                              [1.01694915e+02, 3.14065039e-02], [4.02684564e+01, 2.74462787e-02],
+                              [1.57894737e+02, 2.25603397e-02]]")"
+        }
+
+    """
+    return dd.io.load(path)["madmom_features"]
+
+
+def load_mfcc(path):
+    """Load da_tacos mfcc from a file
+
+    Args:
+        path(str or file-like): File-like object or path to features file
+
+    Returns:
+        np.array: mfcc
+
+    """
+    return dd.io.load(path)["mfcc_htk"]
+
+
+def load_tags(path):
+    """Load da_tacos tags from a file
+
+    Args:
+        path(str or file-like): File-like object or path to features file
+
+    Returns:
+        list: tags
+
+    Examples: [('rock', '0.127'), ('pop', '0.014'), ('alternative', '0.051'), ('indie', '0.048'), ('electronic',
```

Comment:
same here - maybe instead describe the returns as:

```
Returns:
    list: tags, in the form [(tag, confidence), ...]
```

Reply:
done